docs/insiders/faq/sponsoring.md

```diff
@@ -8,25 +8,76 @@ below, ask it on our [discussion board]!
-
-Material for MkDocs always was and will be Open Source, available for free to
-individuals and organizations. As the project grew over time, we found that
-maintaining and managing the overhead that comes with growth became more
-challenging and time-consuming.
-
-In order to sustain the project and add new and useful features more frequently,
-we decided to create the [Insiders] edition, with early access to the latest and
-greatest features of Material for MkDocs. The subscription-based model of the
-Insiders edition allows us to dedicate more time and resources to the project,
-which benefits all
-users of Material for MkDocs. Once our funding goals based on monthly
-subscriptions are hit, the Insiders features of those^ funding goals are released
-to the community edition, letting everyone benefit from them.
-
-Maintaining both the community and Insiders editions is an ongoing process, and
-we rely on our sponsors to support us on a monthly basis, which makes this whole
-project possible.
+[__Why did we create Material for MkDocs Insiders?__](#insiders-edition){ #insiders-edition }
+
+So, here's the deal: Material for MkDocs began as a passion project, which we
+made open-source to share freely with everyone. However, as the project's
+popularity grew over the last few years, so did the demands of its maintenance
+and overhead. Faced with the growing demands and the tough choice between
+discontinuing our efforts or dedicating ourselves full-time, we chose to commit
+ourselves fully to the project.
+
+To sustain this full-time commitment and to expand the team, we introduced
+Material for MkDocs Insiders. This private Insiders repository, available
+exclusively to our monthly sponsors, is where we first release the latest
+advancements of Material for MkDocs. That way our sponsors get the premium
+features in advance of all others users as a thank you for their support. But
+here's the best part: once we achieve specific funding goals, these advancements
+become available to the wider community. This model ensures ongoing development
+and accessibility for all, keeping the project vibrant and up-to-date.
+
+We communicate these funding goals and their progress openly, ensuring our
+community is involved every step of the way. It's our way of making sure that
+while we do need the support to keep going, everyone will eventually benefit
+from the advancements made possible by our sponsors.
+
+[__Why is the Insiders offered as a subscription model?__](#subscription-model){ #subscription-model }
+
+You might wonder why we opted for a subscription model for the Insiders edition
+instead of a one-time purchase. Here's the thing: Material for MkDocs is in
+constant evolution. Our work is never done because there's always something new
+on the horizon – be it adapting to upstream dependency updates, keeping pace
+with environment changes like software or browser updates, or bringing to life
```

Comment:
What software does Material need to be adapted to if not dependencies and browsers? Do you mean operating systems? Unless it is important, I would drop the unspecific "software" bit.